internal/gps/cmd.go

```diff
@@ -22,6 +23,7 @@ import (
 type monitoredCmd struct {
 	cmd     *exec.Cmd
 	timeout time.Duration
+	output  *syncBuffer
```

Comment:
yeah, this seems like it might be overkill, per your note in the OP.

what if instead, if the user calls `monitoredCmd.combinedOutput()`, then we just reuse one of the `activityBuffer`s - e.g., assign `cmd.stdout` to `cmd.stderr`? i can't think of any immediate problems with that...